api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java

```diff
@@ -1306,6 +1310,15 @@ private void enableArchives() {
         addPaperRecordActionsIncludesIfNeeded();
     }
 
+    public void enableChildren() {
+        extensions.add(overallAction(CustomAppLoaderConstants.Extensions.PATIENT_CHILDREN_OVERALL_ACTION,
+                "registration.patient.children.label",
+                "fas fa-fw fa-child",
+                "link",
+                "pihcore/patientRegistration/children.page?patientId={{patient.patientId}}",
```

Comment:
Not 100% sure about nesting this under patientRegistration.  I can see the argument, but still not 100% sure.

Reply:
Yeah, I can see the argument as well, but let's just create a new "pihcore/children" folder instead.